scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -266,22 +266,7 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
         {% if audits | count_cursor > 0 %}
         <tr>
           <td colspan=2>
-            <table id="audit-table" class="table table-sm" style="background-color: transparent">
```

Comment:
Moved to templates/cases/utils so it's used by both case page and case-report page